### ISSUE TITLE
git doesn't work on arm nodes

### ISSUE THREAD

**Issue by kryptn36** [gimlet-io/gimlet#617]
When using an arm node i get the following error:
`utils.go:22: repo-init: fork/exec /usr/bin/git: exec format error`

**Comment by laszlocph** [comment]
What Gimlet version do you use?

We just tested v0.19.0 on a Mac M1 and it worked. (This yaml https://raw.githubusercontent.com/gimlet-io/gimlet/main/deploy/gimlet.yaml)

**Comment by kryptn36** [comment]
I used the latest version on a Hetzner CAX21 Node (ARM64)

Here are are my onechart-values:
```
image:
  repository: ghcr.io/gimlet-io/gimlet
  tag: v0.19.0
ingress:
  host: gimlet.DOMAIN.TLD
  tlsEnabled: true
  ingressClassName: traefik
  annotations:
    traefik.ingress.kubernetes.io/router.entrypoints: websecure
    cert-manager.io/issuer: letsencrypt
containerPort: 9000
nodeSelector:
  type: control-plane
probe:
  enabled: true
  path: /
vars:
  HOST: "https://gimlet.DOMAIN.TLD"
  API_HOST: http://gimlet.default.svc.cluster.local:9000
  FEATURE_POSTHOG: false
resources:
  ignoreLimits: true
volumes:
  - name: data
    path: /var/lib/gimlet-dashboard
    size: 1Gi
```

**Comment by laszlocph** [comment]
I see the issue.
Let me see what can I do.

**Comment by laszlocph** [comment]
#621 
#622

**Comment by laszlocph** [comment]
Will be fixed in v0.19.2, which is due in a day approximately.

**Comment by laszlocph** [comment]
https://github.com/gimlet-io/gimlet/releases/tag/gimlet-v0.19.2 is out. I tested it on CAX21.
